packages/shared/src/styles/base.css

```diff
@@ -102,6 +102,8 @@ body {
 
   --theme-overlay-cabbage-opacity: theme('colors.cabbage.90')A3;
   --theme-overlay-onion-opacity: theme('colors.onion.90')A3;
+  --theme-fade-from: theme('colors.pepper.90');
```

Comment:
Although this value is widely available, it needs to be consistent regardless of theme. Hence, the value is the same on both themes.